src/Turbo/src/TurboResponse.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class TurboResponse extends Response
+{
+    public function __construct(?string $content = '', int $status = 200, array $headers = [])
```

Comment:
```suggestion
    /**
     * @param array<string, string|string[]> $headers
     */
    public function __construct(?string $content = '', int $status = 200, array $headers = [])
```

To please the CI :)

Reply:
Adding this doesn't make sense to me - the parent doesn't declare anything so it doesn't make sense that a child class does it. That'd be random, just to please a tool with no brain.